src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Cart/Blocks/View/order_information.html.twig

```diff
@@ -24,25 +24,35 @@
  *#}
 
 <div class="card" data-role="order-information">
-  <h3 class="card-header">{{ 'Order information'|trans({}, 'Admin.Orderscustomers.Feature') }}</h3>
+  <h3 class="card-header">{{ 'Cart information'|trans({}, 'Admin.Orderscustomers.Feature') }}</h3>
   <div class="card-body">
-    {% if cartView.orderInformation.id %}
-      <h2>
-        <a href="{{ getAdminLink('AdminOrders', true, {'id_order': cartView.orderInformation.id, 'vieworder': 1}) }}">
-          {{ 'Order #%d'|trans({
-            '%d': cartView.orderInformation.id|format('"%06d')
-          }, 'Admin.Orderscustomers.Feature') }}
-        </a>
-      </h2>
-      <p class="mb-0">
-        {{ 'Made on:'|trans({}, 'Admin.Orderscustomers.Feature') }}
-        {{ cartView.orderInformation.placed_date }}
-      </p>
+    <p class="mb-0">
+      {{ 'Created:'|trans({}, 'Admin.Orderscustomers.Feature') }}
```

Comment:
About the date format, I think we have enough space here to have, for example, "November 28, 2023 at 2:29pm" which is easier to read (and to translate). What do you think?

Reply:
I am using the standardized format used across the whole backoffice. Plus, for me, it's easier to read 28.11.2023 than November 28, 2023.

Reply:
Ok but what about translation? I think we had issues in the past because the format is different depending on the country.